packages/vuepress-theme-odyssey/styles/abstracts/_theming.scss

```diff
@@ -0,0 +1,83 @@
+// Copyright 2020-Present Okta, Inc.
```

Comment:
Since this isn't being publicly distributed, we don't need to add the license files here.

Reply:
Removed.